index.html

```diff
@@ -167,16 +167,20 @@ <h3>Goals and Scope</h3>
       
       <section id="background">
         <h3>Background</h3>
-        <p>At the core of the character model is the Universal Character Set (UCS), defined jointly by the <cite>Unicode Standard</cite> [[Unicode]] and ISO/IEC 10646 [[ISO10646]]. In this document, <dfn>Unicode</dfn> is used as a synonym for the Universal Character Set. A successful character model allows Web documents authored in the world's writing systems, scripts, and languages (and on different platforms) to be exchanged, read, and searched by the Web's users around the world.</p>
+        <p><a>Unicode</a>, also known as the <a>Universal Character Set</a>, allows Web documents to be authored in any of the world's writing systems, scripts, or languages, on any computing platforms and then to be exchanged, read, and searched by the Web's users around the world.</p>
         
-        <p>The first few chapters of the <cite>Unicode Standard</cite> [[Unicode]] provide useful background reading. In addition, the <cite>Unicode Collation Algorithm</cite> [[UTS10]] contains a chapter on searching.</p>
+        <p>The first few chapters of the <cite>Unicode Standard</cite> [[Unicode]] provide useful background reading. Also see the <cite>Unicode Collation Algorithm</cite> [[UTS10]], which contains a chapter on searching.</p>
       </section>
       
       <section id="terminology">
         <h3>Terminology and Notation</h3>
-        <p>This section contains terminology and notation specific to this
-          document.</p>
-        <p>Much of the special terminology needed to understand this document is provided by [[CHARMOD-NORM]] and can be found in the <a href="https://www.w3.org/TR/charmod-norm/#terminology">Terminology and Notation</a> section of that document.
+        <p>This section contains terminology and notation specific to this document.</p>
+        
+        <p>Much of the special terminology needed to understand this document is provided by [[CHARMOD-NORM]] and can be found in the <a href="https://www.w3.org/TR/charmod-norm/#terminology">Terminology and Notation</a> section of that document.</p>
+        
+        <p class="definition"><dfn>Corpus</dfn> The natural language text contained by a document or set of documents which the user would like to search.</p>
+        <p class="definition"><dfn>Segmentation</dfn> The process of breaking natural language text up into distinct words and phrases. This often includes operations such as "named entity recognition" (such as recognizing that the three word sequence <kbd>Dr. Jonas Salk</kbd> is a person's name).</p>
+        <p class="definition"><dfn>Stemming</dfn> A process or operation that reduces words to their "stem" or root. For example, the words <kbd>runs</kbd>, <kbd>ran</kbd>, and <kbd>running</kbd> all share the stem <kbd>run</kbd>.</p>
```

Comment:
https://en.wikipedia.org/wiki/Lexeme uses the term lexeme/lemma. I wonder which one is more suitable.